common.tf

```diff
@@ -93,7 +93,8 @@ module "dr2_kms_key" {
       module.ingest_mapper_lambda.lambda_role_arn,
       module.ingest_asset_opex_creator_lambda.lambda_role_arn,
       module.ingest_folder_opex_creator_lambda.lambda_role_arn,
-      module.ingest_upsert_archive_folders_lambda.lambda_role_arn
+      module.ingest_upsert_archive_folders_lambda.lambda_role_arn,
+      module.ingest_start_workflow_lambda.lambda_role_arn
```

Comment:
You don't need to add the lambda to the KMS key. This is  only needed for the lambdas which access S3 or are triggered from encrypted queues.